src/js/utils/event-listener.js

```diff
@@ -0,0 +1,14 @@
+export default class EventListenerMixin {
```

Comment:
Why not keep mixins as pain objects? Used this way, the `prototype` is basically just a bag for the mixin properties. An object would be sufficient.